src/coreclr/tools/Common/CommandLineHelpers.cs

```diff
@@ -74,10 +74,14 @@ public static TargetOS GetTargetOS(string token)
                 return TargetOS.Windows;
             else if (token.Equals("linux", StringComparison.OrdinalIgnoreCase))
                 return TargetOS.Linux;
-            else if (token.Equals("osx", StringComparison.OrdinalIgnoreCase))
-                return TargetOS.OSX;
             else if (token.Equals("freebsd", StringComparison.OrdinalIgnoreCase))
                 return TargetOS.FreeBSD;
+            else if (token.Equals("osx", StringComparison.OrdinalIgnoreCase))
+                return TargetOS.OSX;
+            else if (token.Equals("ios", StringComparison.OrdinalIgnoreCase))
+                return TargetOS.iOS;
+            else if (token.Equals("iossimulator", StringComparison.OrdinalIgnoreCase))
```

Comment:
nit: use switch

Reply:
I don't think you can use switch since it ignores the case (and ToLower/ToUpper is not a good idea).

Reply:
Why is lowering with`ToLowerInvariant` not a god idea?

Reply:
It's extra memory allocation. In this case it's not a hot path but it's still a bit unnecessary.

Reply:
Please fix all instances in this file so that the style is consistent.

Reply:
Done.